### ISSUE TITLE
RGB and BGR confusion??

### ISSUE THREAD

**Issue by naoto0804** [HCIILAB/Scene-Text-Removal#13]
I suspect that the author confuses RGB and BGR format in the synthetic dataset. 
For example, there are a lot of images with the red sky (should be blue or sky-blue), or a human with blue skin. What do you think?

![img_0](https://user-images.githubusercontent.com/7067992/69779312-ef1e9600-11ea-11ea-8fd1-292fd31fccb1.png)

**Comment by janzd** [comment]
I came to the same conclusion when I was checking the images in the dataset. Besides that, most of the images in the test set seem to be included in the training set as well, which obviously shouldn't happen.

**Comment by Zhouzd21** [comment]
Mxnet loads images in terms of function mx.image.imread() in text2.py. The loaded BGR images are transformed to RGB images automatically.
